HM2_task2.py

```diff
@@ -0,0 +1,6 @@
+# 2
+Eleks = ['Mark', 'Anna', 'Diana', 'Alex', "Inga"]
+Toshiba = ['Mari', 'Vika', 'Alex', 'Mark', 'Olya']
+
+Toshiba.append(Eleks)
+print(Toshiba)
```

Comment:
eleks still have employees please fix

Reply:
@bohdandovbsyh, I need to add more employees to "eleks"? I don^t understand?

Reply:
why add? eleks was taken over by Toshiba. So now eleks do not have employees at all